docs/coding-guidelines/interop-guidelines.md

```diff
@@ -13,6 +13,12 @@ We have the following goals related to interop code being used in dotnet/runtime
 - Ensure maximal managed code reuse across different OS flavors which have  the same API but not the same ABI.
    - This is the case for UNIX and addressing it is a work-in-progress (see issue #2137 and section on "shims" below.)
 
+## Submitting Changes
+
+Interop code implicitly defines the native platform dependencies that .NET has. These dependencies are tracked and modeled according to the [Tracking Platform Dependencies design](https://github.com/dotnet/designs/blob/main/accepted/2021/platform-dependencies/platform-dependencies.md). Whenever a PR is submitted that changes interop code, it needs to be reviewed to determine whether a change to the platform dependencies model is required.
+
+**When submitting a change to interop code, mention or add @dotnet/platform-deps-team as a reviewer.** If necessary, update the corresponding `https://github.com/dotnet/core/blob/main/release-notes/<product-version>/runtime-deps.json` file to reflect the dependency change. The scope of dependencies is at the file/package level, not individual functions, so not all interop changes require an update to the model.
```

Comment:
This is busy work. If you want to be mentioned on any changes to interop code, add the platform-deps-team as codeowner for src/libraries/Common/src/Interop to https://github.com/dotnet/runtime/blob/main/.github/CODEOWNERS . You will be automatically mentioned on any changes in this directory.